app/javascript/packs/components/MainPage.jsx

```diff
@@ -12,9 +38,42 @@ const MainPage = () => {
               <img src="/logo.png" />
               <Text>Welcom to LetsTalk!</Text>
             </LogoTitle>
-            <LoginInput xl={12} placeholder="Login" />
-            <LoginInput xl={12} placeholder="Password" />
-            <LoginButton>Log in</LoginButton>
+            <Formik
+              initialValues={{ login: '', password: '' }}
+              validate={validateValues}
+              onSubmit={onSubmit}
+            >
+              {({ isSubmitting, values, handleChange }) => (
+                <Form>
+                  <Row>
+                    <LoginInput
```

Comment:
Minor: this can be achieved with something like this:
`<Field name='login' label='Email' as={LoginInput} />`

Reply:
In jsx for attributes preferable to use double quotes =)
<Field name="login" label="Email" as={LoginInput} />